src/Helper/LabelHelper.php

```diff
@@ -28,12 +28,15 @@ public static function getPositions($start)
             /** @noinspection PhpMissingBreakStatementInspection */
             case 1:
                 $aPositions[] = 1;
+                break;
             /** @noinspection PhpMissingBreakStatementInspection */
             case 2:
                 $aPositions[] = 2;
+                break;
```

Comment:
Als je voor 2 kiest, moet hij ook 3 en 4 kiezen. Dat er geen break; staat, is voor een reden.